.github/workflows/sng-ingestor-shared-workflow.yml

```diff
@@ -89,6 +95,16 @@ jobs:
         with:
           service-account-key: ${{ secrets.gcloud-auth-staging }}
 
+      - name: Notify Slack if staging fails
+        uses: extenda/actions/slack-notify@v0
+        if: failure()
```

Comment:
It should be skipped when the branch is not "master"

Reply:
Done